src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1901,6 +1901,15 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_RequestSEDActiv
 
     return err;
 }
+
+template <class ImplClass>
+CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_RequestSEDCheckinMsg()
+{
+    ChipLogDetail(DeviceLayer, "TODO : Sending Checking Message");
+
+    // TODO #24243 : Implement Checking message
```

Comment:
Why is the right place for this the connectivity manager?  What sort of message will this be sending?  If it's a message from some higher-level protocol, chances are this needs to live at a higher level.

Reply:
I don't think we have an answer for that as it stands. the "entity" responsible of triggering a check-in message hasn't been fully defined yet. 
By having the trigger function in the connectivity manager, we can easily implement and test the check-in message. Once that's done, we can move the starting point of the chain in it's final location.

Reply:
OK.  In that case, can we add comments on these temporary APIs that they are temporary?